docs/reference/analysis/analyzers/standard-analyzer.asciidoc

```diff
@@ -1,26 +1,47 @@
 [[analysis-standard-analyzer]]
 === Standard Analyzer
 
-An analyzer of type `standard` is built using the
-<<analysis-standard-tokenizer,Standard
-Tokenizer>> with the
-<<analysis-standard-tokenfilter,Standard
-Token Filter>>,
-<<analysis-lowercase-tokenfilter,Lower
-Case Token Filter>>, and
-<<analysis-stop-tokenfilter,Stop
-Token Filter>>.
-
-The following are settings that can be set for a `standard` analyzer
-type:
+The `standard` analyzer wraps the <<analysis-standard-tokenizer,`standard`
+>> tokenizer and the <<analysis-standard-tokenfilter,`standard`>>, 
+<<analysis-lowercase-tokenfilter,`lowercase`>>, and 
+<<analysis-stop-tokenfilter,`stop`>> token filters. The `standard` tokenizer 
+splits tokens on whitespace and drops most punctuation. Periods that are not 
```

Comment:
it doesn't split on white space. it tries to split on word boundaries, as defined by Unicode Text Segmentation algorithm, as specified in Unicode Standard Annex #29..  It is a good tokenizer for most languages (although certain Asian languages could benefit from the ICU tokenizer)